kinetic/distribution.yaml

```diff
@@ -4633,6 +4633,17 @@ repositories:
       type: git
       url: https://github.com/inomuh/indoor_localization.git
       version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/inomuh/indoor_localization-release.git
+      version: 0.0.1-0
```

Comment:
`0.0.1-0` is an unusual version number. I'm pretty sure this will work fine as is, so you don't need to change anything. You might consider starting at `0.1.0` as recommended by [semver](https://github.com/semver/semver/blob/master/semver.md#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase).